PhysicsTools/PatAlgos/python/tools/jetTools.py

```diff
@@ -684,10 +684,9 @@ def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSou
             if 'ParticleTransformerAK4TagInfos' in btagInfo:
                 svUsed = svSource
                 if btagInfo == 'pfNegativeParticleTransformerAK4TagInfos':
-                    svUsed = cms.InputTag(btagPrefix+'inclusiveCandidateNegativeSecondaryVertices'+labelName+postfix)
-                    flip = True 
+                    svUsed, flip, max_sip3dsig_for_flip = cms.InputTag(btagPrefix+'inclusiveCandidateNegativeSecondaryVertices'+labelName+postfix), True, 10.
                 else:
-                    flip = False
+                    svUsed, flip, max_sip3dsig_for_flip = svSource, False, -1.
```

Comment:
would it be possible to take the change and change `flip` to something that has more meaning ?

Reply:
`flip` seems pretty intuitive for me.  Would you prefer `negativeTag`?
Btw, is it worth trying to do this in a more automatized way?  In other words, just take the TagInfo string and if it contains "Negative", then we modify `flip` and `svUsed`.  Instead of doing this for every tagger.

Reply:
@vlimant regarding to the `flip` for the sign flip for negativeTag
Hi @mandrenguyen I think we were adding in sequence, it is possible to unified.

Reply:
Hi @vlimant `flip` here refers the sign flip for the negative tagger.
@mandrenguyen  regarding to the automized way  for unified the change for `flip` info, since the `flip` are not always the same due to the tagger features, we would like to keep the sequence with adding the new tagger